sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ErrorParserSuite.scala

```diff
@@ -41,6 +41,17 @@ class ErrorParserSuite extends SparkFunSuite {
     }
   }
 
+  def intercept(sql: String, messages: String*): Unit = {
+    val e = intercept[ParseException](CatalystSqlParser.parsePlan(sql))
+
+    // Check messages.
+    val error = e.getMessage
+    messages.foreach { message =>
+      assert(error.contains(message))
+    }
+  }
```

Comment:
Please avoid duplications. You can reuse `interceptParseException` of `AnalysisTest`. Please check how the other test suite declare its own intercept by using it.